projects/addon-editor/tokens/editor-options.ts

```diff
@@ -8,13 +8,15 @@ import {
 } from '@taiga-ui/addon-editor/constants';
 
 export interface TuiEditorOptions {
+    readonly emitUpdateTime: number;
     readonly blankColor: string;
     readonly colors: ReadonlyMap<string, string>;
     readonly fontOptions: typeof tuiDefaultFontOptionsHandler;
     readonly linkOptions?: TuiEditorLinkOptions;
 }
 
 export const TUI_EDITOR_DEFAULT_OPTIONS: TuiEditorOptions = {
+    emitUpdateTime: 500,
```

Comment:
I think that it is too much for default value.

I think it should be `300` or even `100`.